src/BenchmarksApps/Kestrel/PlatformBenchmarks/Data/RawDb.cs

```diff
@@ -49,13 +56,14 @@ public async Task<World[]> LoadMultipleQueriesRows(int count)
             {
                 await db.OpenAsync();
 
-                var (cmd, idParameter) = CreateReadCommand(db);
-                using (cmd)
+                using (var cmd = CreateReadCommand())
                 {
+                    cmd.Connection = db;
+                    var param = cmd.Parameter;
```

Comment:
In https://github.com/npgsql/npgsql/issues/3200#issuecomment-708936165 you say

> The typical scenario of reexecuting the same command with the same SQL also does it on the same connection, in which case explicit preparation is the right choice and bypasses everything.

Since this is executed 20 times just changing the value, should it do a prepare here?

Reply:
Should definitely give it a try... I don't pay enough attention as I should to the non-Fortunes benchmarks.

Automatic preparation still has the advantage of doing one less roundtrip - the first execution prepared and executes in the same go, where with explicit preparation they're split. But as you add more executions for a single initial prepare the impact of that goes down.

Reply:
Ye olde score multiplier for composite scores
![image](https://user-images.githubusercontent.com/1142958/96113116-94ae2700-0edb-11eb-8816-f2fcacca24db.png)

> Automatic preparation still has the advantage of doing one less roundtrip - the first execution prepared and executes in the same go, where with explicit preparation they're split. But as you add more executions for a single initial prepare the impact of that goes down.

Which is kinda why I want to bypass the parse; then its auto prep + parse once, rather than parse 20 times with auto prep?

Reply:
OK, I'll try to take some time this weekend to play around with bypassing the parse. In any case I need to run (and update) the benchmarks for the newest Npgsql 5.0.0 (just released preview1) - will look into the parsing thing as part of that.